src/core/Notification/Notification.tsx

```diff
@@ -0,0 +1,188 @@
+  actionElementWrapper: `${baseClassName}_action-element-wrapper`,
+};
+
+export interface NotificationProps extends HtmlDivWithRefProps {
+  /** Style variant. Affects color and icon used.
+   * @default 'neutral'
+   */
+  status?: 'neutral' | 'success' | 'error';
+  /** Set aria-live mode for the Notification text content and label.
+   * @default 'polite'
+   */
+  ariaLiveMode?: 'polite' | 'assertive' | 'off';
+  /** Label for the Notification */
+  labelText?: string;
+  /** Main content of the Notification */
+  children?: ReactNode;
+  /** Elements to be rendered under the notification text. Can be e.g. buttons, links etc. */
+  actionElements?: ReactNode;
+  /** Header variant for Notification */
+  headingVariant?: hLevels;
+  /** Text to label the close button. Visible + `aria-label` in regular size and only used as `aria-label` in small screen variant */
+  closeText: string;
+  /** Click handler for the close button */
+  onCloseButtonClick?: () => void;
+  /** Custom props passed to the close button */
+  closeButtonProps?: Omit<HtmlButtonProps, 'onClick' | 'aria-label'>;
+  /** Use small screen styling */
+  smallScreen?: boolean;
+  /** LabelText is header of component */
+}
+interface InnerRef {
+  forwardedRef: React.RefObject<HTMLDivElement>;
+}
+
+class BaseNotification extends Component<NotificationProps & InnerRef> {
+  render() {
+    const {
+      className,
+      ariaLiveMode = 'polite',
+      status = 'neutral',
+      labelText,
+      children,
+      id,
+      actionElements,
+      closeText,
+      onCloseButtonClick,
+      smallScreen,
+      closeButtonProps = {},
+      headingVariant,
+      ...passProps
+    } = this.props;
+
+    const {
+      className: customCloseButtonClassName,
+      'aria-describedby': closeButtonPropsAriaDescribedBy,
+      ...closeButtonPassProps
+    } = closeButtonProps;
+    const variantIcon = status === 'neutral' ? 'info' : 'error';
+    const buttonAriaLabelSmallScreen = smallScreen
+      ? getConditionalAriaProp('aria-label', [closeText])
+      : false;
+
+    return (
+      <HtmlDivWithRef
+        as="section"
+        {...passProps}
+        className={classnames(
+          baseClassName,
+          `${baseClassName}--${status}`,
+          className,
+          {
+            [notificationClassNames.smallScreen]: !!smallScreen,
+          },
+        )}
+      >
+        <HtmlDiv className={notificationClassNames.styleWrapper}>
+          <HtmlDiv className={notificationClassNames.iconWrapper}>
+            <Icon icon={variantIcon} className={notificationClassNames.icon} />
+          </HtmlDiv>
+
+          <HtmlDiv
+            className={notificationClassNames.textContentWrapper}
+            id={id}
+            aria-live={ariaLiveMode}
+          >
+            <HtmlDiv className={notificationClassNames.content}>
+              {labelText && (
+                <Heading
+                  variant={headingVariant || 'h2'}
```

Comment:
We want to be consistent and give the default values where we deconstruct the props, like done for `ariaLiveMode` and `status` props currently in this file.

Reply:
Fixed. Moved to deconstruct.